config.ru

```diff
@@ -1,6 +1,6 @@
 require 'bundler'
 Bundler.require
 
-require './lib/app.rb'
+require_relative './app/lib/obituary_microservice.rb'
 
-run Talker
+run ObituaryMicroservice
```

Comment:
This is the actual file that runs the service. If you want to try it run: `bundle exec rackup` from the terminal then go to the localhost port specified in your terminal. Make sure to put the NYTimes API key in your bash profile first though.

Reply:
Nice, thanks for the instructions! I might end up just waiting for it to be on heroku since bash profile is still pretty intimidating 😅

Reply:
Thanks Jack!